src/client/dfuse/ops/ioctl.c

```diff
@@ -281,19 +276,24 @@ void dfuse_cb_ioctl(fuse_req_t req, fuse_ino_t ino, unsigned int cmd, void *arg,
 	uid_t			uid;
 	gid_t			gid;
 
-	DFUSE_TRA_INFO(oh, "ioctl cmd=%#x", cmd);
-
 	if (cmd == TCGETS) {
 		DFUSE_TRA_DEBUG(oh, "Ignoring TCGETS ioctl");
 		D_GOTO(out_err, rc = ENOTTY);
 	}
 
+	if (cmd == TIOCGPGRP) {
+		DFUSE_TRA_DEBUG(oh, "Ignoring TIOCGPGRP ioctl");
+		D_GOTO(out_err, rc = ENOTTY);
+	}
+
 	/* Check the IOCTl type is correct */
 	if (_IOC_TYPE(cmd) != DFUSE_IOCTL_TYPE) {
-		DFUSE_TRA_INFO(oh, "Real ioctl support is not implemented");
+		DFUSE_TRA_INFO(oh, "Real ioctl support is not implemented cmd=%#x", cmd);
```

Comment:
Not moving this one to 'DEBUG'?

Reply:
No, if we see users trying this one then we want to check the ioctl to see if it's one we should support or not.